Panic during container estimation announcement

neofs-node 0.27.0-rc.1-1-g3d5eb2a3

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0xd5eeca]

goroutine 138183 [running]:
github.com/nspcc-dev/neofs-node/pkg/services/container/announcement/load/route.CheckRoute(0x11a06c0, 0xc00044c8b0, 0xf, 0xc00230a108, 0x3e8, 0xc003d15d60, 0x2, 0x2, 0x0, 0x0)
        github.com/nspcc-dev/neofs-node/pkg/services/container/announcement/load/route/util.go:27 +0x6a
main.(*usedSpaceService).processLoadValue(0xc00045c940, 0x11b9178, 0xc005bcf260, 0xf, 0xc00230a108, 0x3e8, 0xc003d15d60, 0x2, 0x2, 0x11ad628, ...)
        github.com/nspcc-dev/neofs-node/cmd/neofs-node/container.go:527 +0x1b4
main.(*usedSpaceService).AnnounceUsedSpace(0xc00045c940, 0x11b9178, 0xc005bcf260, 0xc00230a090, 0x11aa298, 0xc00230a0a8, 0xc0005198a0)
        github.com/nspcc-dev/neofs-node/cmd/neofs-node/container.go:471 +0x50b
github.com/nspcc-dev/neofs-node/pkg/services/container.(*responseService).AnnounceUsedSpace.func1(0x11b9178, 0xc005bcf260, 0xfcfda0, 0xc00230a090, 0x0, 0x0, 0x40e4f8, 0x8)
        github.com/nspcc-dev/neofs-node/pkg/services/container/response.go:107 +0x69
github.com/nspcc-dev/neofs-node/pkg/services/util/response.(*Service).HandleUnaryRequest(0xc00030a100, 0x11b9178, 0xc005bcf260, 0xfcfda0, 0xc00230a090, 0xc000519920, 0x0, 0x11c3aa8, 0x11b4698, 0x7f2e60e9e130)
        github.com/nspcc-dev/neofs-node/pkg/services/util/response/unary.go:13 +0x5e
github.com/nspcc-dev/neofs-node/pkg/services/container.(*responseService).AnnounceUsedSpace(0xc000116810, 0x11b9178, 0xc005bcf260, 0xc00230a090, 0xc00230a090, 0xc00230a098, 0xc001a59360)
        github.com/nspcc-dev/neofs-node/pkg/services/container/response.go:105 +0x85
github.com/nspcc-dev/neofs-node/pkg/services/container.(*signService).AnnounceUsedSpace.func1(0x11b9178, 0xc005bcf260, 0xfcfda0, 0xc00230a090, 0xc00230a090, 0x30, 0xf49ee0, 0x7f2e609e0301)
        github.com/nspcc-dev/neofs-node/pkg/services/container/sign.go:123 +0x69
github.com/nspcc-dev/neofs-node/pkg/services/util.(*SignService).HandleUnaryRequest(0xc0001a61a0, 0x11b9178, 0xc005bcf260, 0xfcfda0, 0xc00230a090, 0xc000519ad0, 0x109e3f0, 0x0, 0xc001a6f3c0, 0x18, ...)
        github.com/nspcc-dev/neofs-node/pkg/services/util/sign.go:199 +0x32f
github.com/nspcc-dev/neofs-node/pkg/services/container.(*signService).AnnounceUsedSpace(0xc000116828, 0x11b9178, 0xc005bcf260, 0xc00230a090, 0x0, 0xf9a920, 0xc005bcf200)
        github.com/nspcc-dev/neofs-node/pkg/services/container/sign.go:121 +0x94
github.com/nspcc-dev/neofs-node/pkg/network/transport/container/grpc.(*Server).AnnounceUsedSpace(0xc00044c900, 0x11b9178, 0xc005bcf260, 0xc001a6f380, 0xc00044c900, 0xc005bcf260, 0xc0010b2ba0)
        github.com/nspcc-dev/neofs-node/pkg/network/transport/container/grpc/service.go:127 +0xa4
github.com/nspcc-dev/neofs-api-go/v2/container/grpc._ContainerService_AnnounceUsedSpace_Handler(0xf9a920, 0xc00044c900, 0x11b9178, 0xc005bcf260, 0xc0027749c0, 0x0, 0x11b9178, 0xc005bcf260, 0xc001e88800, 0x1f4)
        github.com/nspcc-dev/neofs-api-go/v2@v2.11.0-pre.0.20211201134523-3604d96f3fe1/container/grpc/service_grpc.pb.go:367 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0002c8540, 0x11c2cf8, 0xc000646d80, 0xc0018790e0, 0xc0004517a0, 0x184fe10, 0x0, 0x0, 0x0)
        google.golang.org/grpc@v1.41.0/server.go:1279 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc0002c8540, 0x11c2cf8, 0xc000646d80, 0xc0018790e0, 0x0)
        google.golang.org/grpc@v1.41.0/server.go:1608 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc0000ac390, 0xc0002c8540, 0x11c2cf8, 0xc000646d80, 0xc0018790e0)
        google.golang.org/grpc@v1.41.0/server.go:923 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
        google.golang.org/grpc@v1.41.0/server.go:921 +0x1fd
```

--- a/pkg/services/container/announcement/load/route/deps.go
+++ b/pkg/services/container/announcement/load/route/deps.go
@@ -29,6 +29,10 @@ type Builder interface {
 	// Empty passed list means being at the starting point of the route.
 	//
 	// Must return empty list and no error if the endpoint of the route is reached.
+	// If there are more than one point to go and the last passed point is included
+	// in that list (means that point is the last point in one of the route groups),
+	// returned route must contain nil point that should be interpreted as signal to,
+	// among sending to other route points, save the announcement in that point.
 	NextStage(a container.UsedSpaceAnnouncement, passed []ServerInfo) ([]ServerInfo, error)
 }
 

--- a/pkg/services/container/announcement/load/route/util.go
+++ b/pkg/services/container/announcement/load/route/util.go
@@ -24,6 +24,15 @@ func CheckRoute(builder Builder, a container.UsedSpaceAnnouncement, route []Serv
 		found := false
 
 		for j := range servers {
+			if servers[j] == nil {
+				// nil route point means that
+				// (i-1)-th node in the route
+				// must, among other things,
+				// save the announcement to its
+				// local memory
+				continue
+			}
+
 			if bytes.Equal(servers[j].PublicKey(), route[i].PublicKey()) {
 				found = true
 				break

